src/Microsoft.Extensions.Configuration.AzureAppConfiguration/IConfigurationRefresher.cs

```diff
@@ -8,12 +8,17 @@
 namespace Microsoft.Extensions.Configuration.AzureAppConfiguration
 {
     /// <summary>
-    /// An interface used to trigger an update for the data registered for refresh with the configuration store.
+    /// An interface used to trigger an update for the data registered for refresh with App Configuration.
     /// </summary>
     public interface IConfigurationRefresher
     {
         /// <summary>
-        /// Refreshes the data from the configuration store asynchronously.
+        /// Uri of the App Configuration endpoint.
```

Comment:
Could just shorten it to "The App Configuration endpoint." The endpoint itself is a URI.